app/(main)/markets/Liquidity.tsx

```diff
@@ -257,6 +255,19 @@ export const Liquidity = ({ id }: { id: Address }) => {
   };
 
   const activeLiquidityOperationState = liquidityOperationState[liquidityOperation];
+  const isEnoughBalance = +amount <= +formatEther(activeLiquidityOperationState.balance);
+  const isButtonDisabled = !amount || !isEnoughBalance;
+
+  const getButtonLabel = () => {
+    const tokenSymbolText =
+      liquidityOperation === LiquidityOperation.ADD
+        ? liquidityOperationState[liquidityOperation].inToken.symbol ?? 'pool tokens'
+        : 'pool tokens';
+
+    if (!amount) return 'Enter amount';
+    else if (!isEnoughBalance) return `Insufficient ${tokenSymbolText} balance`;
+    else return activeLiquidityOperationState.actionTitle;
```

Comment:
since we have return inside each if, we can remove the else

Reply:
`hasEnoughBalance` sounds better IMO

```suggestion
    else if (!hasEnoughBalance) return `Insufficient ${tokenSymbolText} balance`;
    else return activeLiquidityOperationState.actionTitle;
```